packages/vite/src/node/plugin.ts

```diff
@@ -121,7 +121,12 @@ export interface Plugin extends RollupPlugin {
     this: PluginContext,
     source: string,
     importer: string | undefined,
-    options: { custom?: CustomPluginOptions; ssr?: boolean }
+    options: {
+      custom?: CustomPluginOptions
+      ssr?: boolean
+      // @internal
```

Comment:
```suggestion
      /**
       * @internal
       */
```

I tested this locally, and it seems that this needs to be in jsdoc for the types to be stripped